theme/src/css/_zendesk-variables.scss

```diff
@@ -114,7 +114,7 @@ $nav-link-padding-x: 1.5rem !default; // 24px
 $nav-underline-link-highlight-left: 0 !default; 
 $nav-underline-link-highlight-right: 1.5rem !default; // 24px
 
-$navbar-height: 4rem !default;
+$navbar-height: 3.5rem !default;
 $navbar-padding-x: 0 !default;
 
 /* ---------- Custom variables ---------- */
```

Comment:
Can we change all the comments to using the SASS comments if that is the desired format for consistency?

Reply:
Oops, missed those!